documentation/src/main/asciidoc/topics/con_restarting_clusters.adoc

```diff
@@ -1,10 +1,14 @@
 [id='cluster-restarts_{context}']
 = {brandname} cluster restarts
 
-When you bring {brandname} clusters back online after shutting them down, you should wait for the cluster to be available before adding or removing nodes or modifying cluster state.
+When you bring {brandname} clusters back online after using the `shutdown cluster` command, all nodes and caches in the cluster will be unavailable until all nodes rejoin.
+To prevent inconsistencies or data loss, {brandname} restricts access to the data stored in the cluster and modifications of the cluster state until the cluster is fully operational again.
+Additionally, {brandname} disables cluster rebalancing and prevents file stores purging on startup.
+
+{brandname} logs messages for each node during the cluster recovery process, indicating which nodes are available and which are still missing. +
+You can monitor availability of caches using the {brandname} Console or REST API.
 
 If you shutdown clustered nodes with the [command]`shutdown server` command, you must restart each server in reverse order. +
 For example, if you shutdown `server1` and then shutdown `server2`, you should first start `server2` and then start `server1`.
```

Comment:
This is definitively not recommended but might be used as workaround (for now) to have the old behaviour.
In this case the persistence must be set to NOT purge on startup - this will show a warn message as it could cause inconsistence, as well we plan to remove that option later so shared=false setting will be automatically purge on startup!

Reply:
This info was already present in the docs. I don't have the historical context here so Idk if we should keep or not.
I added some info about shutting down individual servers and unshared stores.

Reply:
It was not recommended before, but there was no choice for the customer.
Now they should use 'cluster shutdown' if the unshared filestore should be kept.
For further versions we remove the possibility `unshared=true purge=false` to prevent from inconsistency.
Starting in reverse order is better but does not prevent from stale entries!

Reply:
@wfink So what is your suggestion here? I'm considering removing this part about restarting servers in a reverse order. If you say it's not recommended we should advise users to use cluster shutdown if they want to preserve the unshared file stores after restart. wdyt?

Reply:
I would prefer to keep it even if not recommended it was the only way before having the correct implemented "shutdown cluster" and restart it.
The strong recommendation is to use `purge=true` in case of unshared stores which mean you need to use the shutdown cluster command to not start with an empty cache!
Note that the cluster shutdown works as well (and not purge) if there is only one node, but it will work consistent and don't have inconsistencies